cinema/views.py

```diff
@@ -167,6 +188,18 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by date (ex. ?date=2024-30-10)",
```

Comment:
The OpenAPI parameter 'date' is defined with a type of list, but it should be a string since the filter is applied to a single date string. Update the type to `{'type': 'string'}`.